src/main/java/ecommerce/dao/impl/jdbc/OrderDaoJdbcImpl.java

```diff
@@ -0,0 +1,185 @@
+package ecommerce.dao.impl.jdbc;
+
+import ecommerce.dao.interfaces.OrderDao;
+import ecommerce.exceptions.DataProcessingException;
+import ecommerce.lib.Dao;
+import ecommerce.model.Order;
+import ecommerce.model.Product;
+import ecommerce.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class OrderDaoJdbcImpl implements OrderDao {
+    @Override
+    public List<Order> getUserOrders(Long userId) {
+        String query = "SELECT * FROM orders WHERE user_id = ? AND deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement
+                         = connection.prepareStatement(query)) {
+            statement.setLong(1, userId);
+            try (ResultSet resultSet = statement.executeQuery()) {
+                List<Order> allOrders = new ArrayList<>();
+                while (resultSet.next()) {
+                    Order order = getOrderFromResultSet(resultSet, connection);
```

Comment:
you're not closing statement here 
Your statement will closed when you exit your first TWR block, cause you passed statement there
and I don't understand why you open second TWR with ResultSet

Reply:
fixed